### ISSUE TITLE
onSelect or onClick

### ISSUE THREAD

**Issue by vahidhiv** [agarzola/jQueryAutocompletePlugin#47]
Hi, firstable i appreciate your effort on creating this great plugin,
i want to add onClick or onSelect event listener on the input which uses autocomplete, is it possible? Would you pls help me?
Thank you.

**Comment by agarzola** [comment]
Hello, @vahidhiv. I appreciate the kind words.

I did not create this plugin. It was created by Jörn Zaefferer. It has since been deprecated in favor of [jQuery UI’s Autocomplete](https://jqueryui.com/autocomplete/). Many years ago, I needed to use features on this deprecated plugin that weren’t easy to do with the new version linked above, so I forked it and added some features to it to fit my needs. However, I haven’t used or maintained it in years, so it’s probably better to use something else.

For your own sake, I strongly suggest you use the more up to date [jQuery UI Autocomplete](https://jqueryui.com/autocomplete/).

---
Having said all of the above, I don’t see why you couldn’t bind an event handler on the input separately. Like so:
```javascript
$('#inputID').on('click', someFunction);
```

Perhaps I’m misunderstanding your request, though.
